src/app.js

```diff
@@ -0,0 +1,44 @@
+require('dotenv').config();
+
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const UserRoutes = require('./routes/UserRoutes');
+
+const app = express();
+app.use(bodyParser.json());
+
+
+// Test .env
+console.log('MONGODB_URI:', process.env.MONGODB_URI);
```

Comment:
should split MONGODB_URI to 
- DB_NAME
- DB_USER
- DB_PASSWORD
- DB_HOST
- DB_PORT